demonstrations/tutorial_vqe_bond_dissociation.py

```diff
@@ -0,0 +1,389 @@
+forming of bonds. They are characterized by an energy barrier that determines
+the likelihood that the reaction occurs. These energy landscapes formed by these barriers are the
+key to understanding how chemical reactions occur, at the deepest possible level.
+
+.. figure:: /demonstrations/vqe_bond_dissociation/reaction.png
+    :width: 50%
+    :align: center
+
+    An example chemical reaction
+
+In this tutorial, you will learn how to use PennyLane to simulate chemical reactions by
+constructing potential energy surfaces for molecular transformations. In the process, you will
+learn how quantum computers can be used to calculate equilibrium bond lengths, activation energy
+barriers, and reaction rates. As illustrative
+examples, we use tools implemented in PennyLane to study diatomic bond dissociation and reactions
+involving the exchange of hydrogen atoms.
+
+
+
+##############################################################################
+
+Potential Energy Surfaces 
+---------------------------------------------------------------------
+
+`Potential energy surfaces (PES) <https://en.wikipedia.org/wiki/Potential_energy_surface>`_
+describe the equilibrium energy of molecules for different positions of
+its atoms. The concept originates from the fact that nuclei are much heavier than
+electrons and thus move more slowly, allowing us to decouple their motion. We can then solve for
+the electronic wavefunction with the nuclei clamped to their respective positions. This leads
+to a separation of the nuclear and electronic parts of the Schrödinger equation, meaning we only
+need to solve the electronic equation:
+
+.. math:: H(R)|\Psi \rangle =  E|\Psi\rangle.
+
+From this perspective arises the concept of the electronic energy of a molecule, :math:`E(R)`,
+as a function of nuclear coordinates :math:`R`. The energy :math:`E(R)` is the expectation value
+of the molecular Hamiltonian, :math:`E(R)=\langle \Psi_0|H(R)|\Psi_0\rangle`, taken
+over the ground state :math:`|\Psi_0(R)`. The potential energy surface is
+precisely this function :math:`E(R)`, which connects energies to different geometries of the
+molecule. It gives us a visual tool to understand chemical reactions by associating
+stable molecules (reactants and products) with local minima, transition states with peaks,
+and by identifying the possible routes for a chemical reaction to occur.
+
+To build the potential energy surface, we compute the energy for fixed positions of the nuclei,
+and subsequently move the nuclei in incremental steps, computing the new energies at each point.
+The obtained set of energies corresponds to a grid of nuclear positions and the plot of
+:math:`E(R)` gives rise to the potential energy surface.
+
+
+.. figure:: /demonstrations/vqe_bond_dissociation/pes.png
+    :width: 75%
+    :align: center
+
+    Illustration of a potential energy surface for a diatomic molecule
```

Comment:
```suggestion
    Illustration of a potential energy surface for a diatomic molecule.
```
Should there be periods after captions? (I added one to the first graphic too, ignore if not necessary)